js_modules/dagster-ui/packages/ui-core/src/runs/LogsScrollingTableNew.tsx

```diff
@@ -0,0 +1,107 @@
+
+  const virtualizer = useVirtualizer({
+    count: filteredNodes.length,
+    getScrollElement: () => parentRef.current,
+    estimateSize: () => 64,
+    overscan: 20,
+    paddingEnd: 40,
+  });
+
+  const totalHeight = virtualizer.getTotalSize();
+  const items = virtualizer.getVirtualItems();
+
+  // Determine whether the user has scrolled away from the bottom of the log list.
+  // If so, we no longer pin to the bottom of the list as new logs arrive. If they
+  // scroll back to the bottom, we go back to pinning.
+  useEffect(() => {
+    const parent = parentRef.current;
+
+    const onScroll = () => {
+      const totalHeight = virtualizer.getTotalSize();
+      const rectHeight = virtualizer.scrollRect.height;
+      const scrollOffset = virtualizer.scrollOffset;
+
+      // If we're within a certain threshold of the maximum scroll depth, consider this
+      // to mean that the user wants to pin scrolling to the bottom.
+      const maxScrollOffset = totalHeight - rectHeight;
+      const shouldPin = scrollOffset > maxScrollOffset - BOTTOM_SCROLL_THRESHOLD_PX;
+
+      pinToBottom.current = shouldPin;
+    };
+
+    parent && parent.addEventListener('scroll', onScroll);
+    return () => {
+      parent && parent.removeEventListener('scroll', onScroll);
+    };
+  }, [virtualizer]);
+
+  // If we should pin to the bottom, do so when the height of the virtualized table changes.
+  useEffect(() => {
+    if (pinToBottom.current) {
+      virtualizer.scrollToOffset(totalHeight, {align: 'end'});
+    }
+  }, [totalHeight, virtualizer]);
+
+  const content = () => {
+    if (logs.loading) {
+      return (
+        <Box margin={{top: 32}}>
+          <ListEmptyState>
+            <NonIdealState icon="spinner" title="Fetching logs..." />
+          </ListEmptyState>
+        </Box>
+      );
+    }
+
+    return (
+      <Inner $totalHeight={totalHeight}>
+        <DynamicRowContainer $start={items[0]?.start ?? 0}>
+          {items.map(({index, key}) => {
+            const node = filteredNodes[index]!;
+            const textMatch = textMatchNodes.includes(node);
+            const focusedTimeMatch = Number(node.timestamp) === filter.focusedTime;
+            const highlighted = textMatch || focusedTimeMatch;
+
+            const row =
+              node.__typename === 'LogMessageEvent' ? (
+                <Unstructured node={node} metadata={metadata} highlighted={highlighted} />
+              ) : (
+                <Structured node={node} metadata={metadata} highlighted={highlighted} />
+              );
+
+            return (
+              <div ref={virtualizer.measureElement} key={key} data-index={index}>
+                {row}
+              </div>
+            );
+          })}
+        </DynamicRowContainer>
+      </Inner>
+    );
+  };
+
+  return (
+    <ColumnWidthsProvider>
+      <Headers />
+      <Container ref={parentRef}>{content()}</Container>
+    </ColumnWidthsProvider>
+  );
+};
```

Comment:
Wow much simpler indeed, this is awesome 🙌 

I think there are a couple final tests I'd do before merging this based on things that were kinda tricky in here, but I'm super excited to get rid of the old V1:

- If you make the window narrower, do the rows resize?

- In the old version, the rows are sized to fit the text, BUT there's also a maxHeight, and when the maxHeight is exceeded you get the "Show Full Message" banner.  Probably good to test that case with a jumbo text row and make sure the banner still appears, because I think the CSS was kinda weird and delicate. 

Other than that, let's go!